BUILD_macos.md

```diff
@@ -81,6 +81,11 @@ PRs with corrections and updates are welcome!
   * `brew install node yarn`
   * Currently, [`yarn`](https://classic.yarnpkg.com/en/docs/install) (< 2.0.0) is required
 
+* Yarn uses wasm-pack, so if that's not present, we need to install it
+  ```shell
+  curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+  ```
+
```

Comment:
```suggestion
* Teleport uses [wasm-pack](https://rustwasm.github.io/wasm-pack/),
  so if that's not present, we need to [install](https://rustwasm.github.io/wasm-pack/installer/) it
```

- Yarn doesn't use wasm-pack, Teleport does
- There are several ways to install it, so let's just link to the installation docs.

Reply:
Fair enough, thanks for the input